api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -333,6 +333,46 @@
 			constraintName="key_for_location_tag_map"/>
 	</changeSet>
 	
+
+		<changeSet id="TRUNK-3422-adding-privileges" author="PralayRamteke" >
+			<preConditions  onFail="MARK_RAN">
+   				<not>
+   				<sqlCheck expectedResult="0">
+    				SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
```

Comment:
@dkayiwa  is this what you asked me to do? i removed count(*) != 0 statement. :)